common/src/main/java/com/android/designcompose/common/FigmaDocId.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose.common
+
+/** When version id is not specified, it will load head of the doc. */
+data class FigmaDocId(var id: String, var versionId: String = "") {
```

Comment:
Would it be possible to change this to `DesignDocId`? We should one day support tools other than Figma (or support hand written DCF files), and it would be good if we just added an enum or something to this type when we do that, rather than needing to rename it.